Protein-systematic-characterization

```diff
@@ -0,0 +1 @@
+Subproject commit 51023d9f94532ff6f96340c9c0e52c1cc1dfe6af
```

Comment:
Hmm.. just wondering what the purpose of recording this is? If it's good, I'll let it through.